web/src/textUtils.ts

```diff
@@ -0,0 +1,68 @@
+import { ContentType, Exam } from './types'
+import { TFunction } from 'i18next'
+
+export function headingText(exam: Exam, t: TFunction) {
+  switch (exam) {
+    case Exam.SUKO:
+      return t('header.suko')
+    case Exam.LD:
+      return t('header.ld')
+    case Exam.PUHVI:
+      return t('header.puhvi')
+  }
+}
```

Comment:
Mitäs jos nää oiksi objektei:

```suggestion
export const headingTextByExam = {
  Exam.SUKO: t('header.suko')
  Exam.LD: t('header.ld')
  Exam.PUHVI: t('header.puhvi')
}
```

Jota käytettäis tyylii `headingTextByExam[Exam.SUKO]`

Reply:
Mistä toi t -funktio tulisi tulisi?

Reply:
Vai tehdäänkö custom hookki, niin sit saadaan sen sisällä paljastettua const {t} = useTranslate. Nykyinen tapa oli simppelimpi kun on vain pelkkä funktio jolle voi t -funktion passata

Reply:
Hyvä pointti toi t-funktion saatavuus.

Meillähän on jo `useConstantsWithLocalization`. Ehkä sen vois nimetä uusiks, vaik `useLudosTranslations`, ja nää mun ehdottamat objektit vois laittaa sinne. Ja sit koodissa käytettäis:

```
const lt = useLudosTranslations()
return (<p>${lt.headingTextByExam[Exam.SUKO]}</p>)
```

Edut verrattuna `textUtils.ts`:n nykytilaan:
1) käyttöpaikois tarttee importtaa vain useLudosTranslations, kun nyt pitää importtaa sekä useTranslation että jokainen funkkari erikseen textUtils:tä
2) mukavampi käyttää, ku ei tartte passaa `t`:tä joka funktiokutsulle

Reply:
Hyvä idea! :) tehdään niin